imports/plugins/core/system-info/server/no-meteor/queries/systemInformation.js

```diff
@@ -0,0 +1,16 @@
+import packageJson from "/package.json";
+
+/**
+ * @name queries.systemInformation
+ * @method
+ * @memberof SystemInformation/GraphQL
+ * @summary get systemInformations
+ * @param {Object} context - an object containing the per-request state
+ * @return {<Object} System Information
+ **/
+export default async function systemInformation(context) {
+  return {
+    apiVersion: packageJson.version,
+    plugins: await context.collections.Packages.find({ version: { exists: true } }).toArray()
```

Comment:
@aldeed is this what you meant by await with the `toArray()`? testing through graphql-alpha still returns plugins{}